frontend/src/app/page.tsx

```diff
@@ -6,7 +6,7 @@ export default function Home() {
         <main className={styles.main}>
             <div className={styles.description}>
                 <p>
-                    Get started by editing&nbsp;
+                    Get started  by editing&nbsp;
```

Comment:
an extra space !!

Reply:
Part of my troubleshooting why I couldn't make a PR ;)